content/blog/guides/2020/2020-03-25-hello-world/index.mdx

```diff
@@ -1,16 +1,173 @@
-Oh, and here's a great quote from this Wikipedia on
-[salted duck eggs](http://en.wikipedia.org/wiki/Salted_duck_egg).
+## Introduction
 
-> A salted duck egg is a Chinese preserved food product made by soaking duck
+This guide will help **Frontend Engineers** and people from the **UI Community**
+to develop a common understanding of **collaboration** and why this particular
+skill can improve the process of software development and product design in your
+company. I will **not** describe implementation details or how to create the
+next big Design System. But I will provide you the right methods and practices
+to build a Pattern Library or even a Design System. It will also help people who
+already maintain a Pattern Library. To point out some problems and situations I
+will do some _storytelling_. Let's get started.
+
+## The Status Quo Story
+
+Let’s imagine ourselves in a successful company with a customer-centric
+application. The company has a **product department** with a bunch of
+cross-functional teams. The product designers maintain a **design library** to
+align the design files, which they share with **developers** and **managers**.
+Besides that, the developers maintain a **pattern library**, which provides UI
+components and variables for the code in the product. Let's take a closer look
+at the roles:
+
+### Designer
+
+**Job Title:** Senior Product Designer
+
+**Team:** Unicorn 🦄
+
+**Current Task:** After a meeting with other product designers everyone agreed
+to do small improvements to their design library. That person decided to update
+the design files to a new shiny color palette to improve accessibility.
+
+**Complication:** As soon as the designer updates the design library, we don’t
+have an alignment with our Pattern Library anymore. 😢😢😢
+
+### Developer
+
+**Job Title:** Senior Frontend Engineer
+
+**Team:** Delicious ✨
+
+**Current Task:** Implementation of a new Product Teaser. That person recognized
+some inconsistencies regarding the **colors** in the design file. It is not
+aligned with the **pattern library**. Nevertheless, that person is implementing
+the code solution with those new colors.
+
+**Complication:** There is a misleading impression that lets the developer think
+that this is an exceptional situation. This will justify the code implementation
+of the new colors only for the new product teaser.
+
+### Product Manager
+
+**Job Title:** Product Manager
+
+**Team:** Deliver 📦
+
+**Current Task:** The Product Manager is in a meeting with **product designers**
+and **developers** to discuss the upcoming sprint. To have a basis of discussion
+the product manager prepared user stories and tasks.
+
+**Complication:** Due to the lack of terminology it’s hard to grasp the sense of
+the user stories and tasks for the **designers** and **developers**.
+
+## It's not a Technical Problem
+
+The root of all evil here is the **lack of collaboration**. It’s like they are
```

Comment:
I have to say I do like this paragraph because for the first time I do understand what you wanna tell me. like I can literally feel the pain.